user-service/src/main/java/com/spike/user/helper/UserHelper.java

```diff
@@ -310,9 +310,14 @@ public UserProfilePicture updateUserProfilePicture(MultipartFile profilePicture,
                 String newFileName = user.getEmployeeCode() + "_profile_picture" + fileExtension;
 
                 // Define the file path where the profile picture will be saved
-                String filePath = uploadDir + File.separator + newFileName;
+                String filePath = oldFilePath != null ? oldFilePath : (uploadDir + File.separator + newFileName);
```

Comment:
are er still using multipart, i thought we were going to use base64 now onwards.